CODEOWNERS

```diff
@@ -11,7 +11,7 @@
 
 # scenario directory (Rico AND Justin)
```

Comment:
small thing: could you replace Justin with Robert?

Reply:
Fixed in [6a622d4](https://github.com/GAIA-X4PLC-AAD/ontology-management-base/commit/6a622d4fdd24480f16758d3943524710e5b06740)